packages/components/stories/index.stories.js

```diff
@@ -0,0 +1,19 @@
+import { storiesOf } from '@storybook/react';
```

Comment:
@reza-fs we probably have to keep each story in its folder something like
 
`button
  -index.js
  -torySettings.js
  -examples.js
 blablabla
`

can you please add a proper example so everyone follows the structure.

Reply:
Sure, I'm on it.

Reply:
Added some sample components, please have a look @Mahboobeh-binary